### ISSUE TITLE
Nuxt 3 Support

### ISSUE THREAD

**Issue by harlan-zw** [nuxt-community/vuetify-module#522]
Hey :wave: It looks like this module hasn't been updated for Nuxt 3. In an effort to improve the developer experience of modules, I've updated the module to clarify that it only supports Nuxt 2.

If Nuxt 3 support is added it will be moved to the https://github.com/nuxt-modules organisation.

Alternatives:
- [userquin/vuetify-nuxt-module](https://github.com/userquin/vuetify-nuxt-module)
- [nuxt-alt/vuetify](https://github.com/nuxt-alt/vuetify)
- [diy workaround](https://github.com/nuxt-community/vuetify-module/issues/475#issuecomment-1287885444)

@kevinmarrec Is Nuxt 3 support something you have planned? If not, perhaps we can deprecate this module for the alternative?

Please let us know and thanks for your work!

**Comment by kevinmarrec** [comment]
Hi @harlan-zw, I'm not planning to develop it so, indeed, you can deprecate this module for the alternative regarding Nuxt 3 support.

**Comment by harlan-zw** [comment]
Thank you for the response. Will come up with a plan for deprecation in the next Nuxt team meeting.

**Comment by harlan-zw** [comment]
Hey @userquin, do you have any thoughts on what we should do with this module?

Do you think it's worth moving your code into this repo if you're happy to take on maintenance? We'd move it into @nuxt-modules if so.

Otherwise, we may look to mark this repo as archived at some point.

**Comment by userquin** [comment]
@harlan-zw yeah, I can transfer the repo, I need to fix a few things before the transfer: the current active PR and update i18n integration. Since Vuetify 3 and i18n still under development it will take some time.

**Comment by harlan-zw** [comment]
No rush at all, thanks for the reply! Just trying to figure out a plan for the module. :)

**Comment by TechAkayy** [comment]
My experience with @userquin's https://github.com/userquin/vuetify-nuxt-module has been very positive and currently the best option for Nuxt 3. 

The only downside is that it currently supports only Vite, and not webpack. 

If it comes to the decision of deprecating this package, can I suggest making https://github.com/userquin/vuetify-nuxt-module as the official with the note that its only for Vite, probably webpack support could come from the community later. 

It would be very unfortunate for Nuxt to not have an official module for Vuetify, given the fact that both are huge & popular within the Vue ecosystem.

**Comment by userquin** [comment]
@harlan-zw @TechAkayy I'm starting to get tired of maintaining the nuxt module for vuetify, it doesn't work at all as expected, I'm going to leave it until vuetify is more stable, I'll look for another UI since I can't wait any longer, it's been 1 year and I don't see any progress anytime soon, I cannot spend more time on it (check last PR merged in the repo).

**Comment by TechAkayy** [comment]
Still, thanks bunch @userquin for this amazing module, and for all your help to the Nuxt Vuetify community. 

This provides a great base for any future Nuxt Vuetify maintainers (or) the Vuetify core team to continue supporting Vuetify for Nuxt.

**Comment by userquin** [comment]
@harlan-zw finally moved to vuetifyjs https://github.com/vuetifyjs/nuxt-module and new docs here https://nuxt.vuetifyjs.com/

**Comment by userquin** [comment]
@harlan-zw can you update readme links to the new module (now on vuetifyjs org. repo, readme using my github account) and release a patch version to update npm registry readme file?

EDIT: nuxt docs using npm registry readme file, this way also nuxt docs should also be updated (https://nuxt.com/modules/vuetify)
